workflow/snakemake/inputs.yaml

```diff
@@ -0,0 +1,5 @@
+events: 20000
+fitdata: code/fitdata.C
+gendata: code/gendata.C
+data: data.root
+plot: plot.png
```

Comment:
Here we need to set the full path of these output files, so we need to prepend `results/`